core/src/main/java/pl/agh/edu/ui/component/button/ScenarioButton.java

```diff
@@ -22,55 +25,71 @@
-		setActor(button);
+		setActor(scenarioButton);
 		setSize();
 
 		ScenarioButtonStyle.createPad(this);
 		setResolutionChangeHandler(this::updateSizes);
 
 		createActors();
 		createButton();
+		addSoundEvents();
 		setTouchable(enabled);
 	}
 
 	public void createActors() {
-		titleLabel = createLabel(scenarioButtonStyle.getTitleLabelPath(), scenarioButtonStyle.getTitleLabelFont(), (int) (button.getHeight() / 30));
+		titleLabel = createLabel(scenarioButtonStyle.getTitleLabelPath(), scenarioButtonStyle.getTitleLabelFont(), (int) (scenarioButton.getHeight() / 30));
 		titleLabel.setAlignment(getAlign());
 
 		scenarioImage = new Image(skin.getDrawable(scenarioButtonStyle.getIconPath()));
 
-		descriptionLabel = createLabel(scenarioButtonStyle.getDescriptionLabelPath(), scenarioButtonStyle.getDescriptionLabelFont(), (int) (button.getHeight() / 40));
+		descriptionLabel = createLabel(scenarioButtonStyle.getDescriptionLabelPath(), scenarioButtonStyle.getDescriptionLabelFont(), (int) (scenarioButton.getHeight() / 40));
 		descriptionLabel.setAlignment(getAlign());
 
-		seasonLabel = createLabel(scenarioButtonStyle.getSeasonLabelPath(), scenarioButtonStyle.getDescriptionLabelFont(), (int) (button.getHeight() / 40));
+		seasonLabel = createLabel(scenarioButtonStyle.getSeasonLabelPath(), scenarioButtonStyle.getDescriptionLabelFont(), (int) (scenarioButton.getHeight() / 40));
 		seasonLabel.setAlignment(getAlign());
 	}
 
 	public void createButton() {
-		button.clearChildren();
-		button.add(titleLabel).width(scenarioButtonStyle.getTitleDescriptionWidth()).height(scenarioButtonStyle.getTitleDescriptionHeight()).row();
-		button.add(scenarioImage).width(scenarioButtonStyle.getIconWidthHeight()).height(scenarioButtonStyle.getIconWidthHeight()).padTop(scenarioButtonStyle.getElementPadTop())
+		scenarioButton.clearChildren();
+		scenarioButton.add(titleLabel).width(scenarioButtonStyle.getTitleDescriptionWidth()).height(scenarioButtonStyle.getTitleDescriptionHeight()).row();
+		scenarioButton.add(scenarioImage).width(scenarioButtonStyle.getIconWidthHeight()).height(scenarioButtonStyle.getIconWidthHeight()).padTop(scenarioButtonStyle
+				.getElementPadTop())
 				.center().row();
-		button.add(descriptionLabel).width(scenarioButtonStyle.getTitleDescriptionWidth()).height(scenarioButtonStyle.getTitleDescriptionHeight()).padTop(scenarioButtonStyle
-				.getElementPadTop()).row();
-		button.add(seasonLabel).width(scenarioButtonStyle.getSeasonLabelWidth()).padTop(scenarioButtonStyle.getElementPadTop()).center();
+		scenarioButton.add(descriptionLabel).width(scenarioButtonStyle.getTitleDescriptionWidth()).height(scenarioButtonStyle.getTitleDescriptionHeight()).padTop(
+				scenarioButtonStyle
+						.getElementPadTop()).row();
+		scenarioButton.add(seasonLabel).width(scenarioButtonStyle.getSeasonLabelWidth()).padTop(scenarioButtonStyle.getElementPadTop()).center();
+	}
+
+	private void addSoundEvents() {
+		addListener(new InputListener() {
```

Comment:
lepiej było by dodać to do przycisku bo teraz mamy taką sytuacje że jak klikamy wokół tego przysiska to odpala nam się dzwięk